README.md

```diff
@@ -58,10 +58,10 @@ Here is an overview of all fields that can be present in plugin meta YAML files
 
 |field name in YAML | description | mandatory |
 |-------|-------|-------|
-| id | plugin ID | yes |
+| name | plugin name | yes |
```

Comment:
it makes sense to describe format here. It should be URL compatible now, the same for publisher

Reply:
I added new fields. I didn't discuss URL compatibility constraints before, so leaving that to be contributed by others.